llvm/test/MC/RISCV/rv32zaamo-invalid.s

```diff
@@ -0,0 +1,11 @@
+# RUN: not llvm-mc -triple riscv32 -mattr=+experimental-zaamo < %s 2>&1 | FileCheck %s
```

Comment:
Is this duplicating existing A test cases? If so can we split the A extension tests into 2 files as a pre-commit and add these the new Zaamo and Zalrsc command lines in this patch?